src/redux/Courier/taskSelectors.js

```diff
@@ -17,6 +17,7 @@ export const selectIsTasksRefreshing = state => state.entities.tasks.isRefreshin
 export const selectIsTasksLoadingFailure = state => state.entities.tasks.loadTasksFetchError
 export const selectIsTaskCompleteFailure = state => state.entities.tasks.completeTaskFetchError
 export const selectTaskFilters = state => state.ui.tasks.excludeFilters
+export const selectIsPolylineOn = state => state.ui.tasks.isPolylineOn
```

Comment:
I guess `isPolylineOn` will be `undefined` for the existing users. Is it what we want? Or do we  want it also to be true by default in that case?

Reply:
I think the initial state is set here : https://github.com/coopcycle/coopcycle-app/pull/1678/commits/a577f44278b49bb2afe65a69db844bc8b9eb4410#diff-9f556b044c2b86f230999b3167c6717644249d59e90412788a54f7ded0c24c6aR26 so it will be set to true (I have set to true by default so existing users see the feature)

am I correct?

Reply:
Maybe it's correct, but I suggest to double check that it works that way for the existing users, and that it's not overridden by the react-persist during rehydration: https://github.com/rt2zz/redux-persist#state-reconciler

Reply:
ah ok I see. if it is not correct I have attempted a commit (see below).

however I am not 100% sure on how to test locally? I should build the app on master then switch branch and load this one? through Metro?

thx